test-clients/src/main/java/com/hedera/services/legacy/client/core/GrpcStub.java

```diff
@@ -142,29 +114,4 @@ public TransactionID deleteFile(AccountID payerAccount,
 		}
 
 	}
-
-	public TransactionID deleteAccount(AccountID payer, Key payerKey,
-			AccountID deleteAccount, Key accKey,
-			AccountID transferAccount,
-			AccountID nodeAccount, Map<String, PrivateKey> pubKey2privKeyMap) throws InvalidProtocolBufferException  {
-		Transaction transaction = Common.tranSubmit(() -> {
-			Transaction deleteRequest;
-			try {
-				deleteRequest = Common.buildCryptoDelete(payer, payerKey,
-						deleteAccount, accKey,
-						transferAccount,
-						nodeAccount, pubKey2privKeyMap);
-			} catch (Exception e) {
-				e.printStackTrace();
-				return null;
-			}
-			return deleteRequest;
-		}, cryptoStub::cryptoDelete);
-		if (transaction != null) {
-			return TransactionBody.parseFrom(transaction.getBodyBytes())
-					.getTransactionID();
-		} else {
-			return null;
-		}
-	}
```

Comment:
Is this class still used somewhere?

Reply:
Yes the `SmartContractTestBitcarbon` still looks like it's being run (I created an issue to rewrite it [here](https://github.com/hashgraph/hedera-services/issues/1555))...once that is done, we can revisit the dead code cleanup, and it might be possible to remove everything. 😄